settings.gradle.kts

```diff
@@ -47,3 +58,21 @@ if (gradle.startParameter.taskNames.contains("updateDependencyLocks") ||
 for (prj in rootProject.children) {
   prj.buildFileName = "${prj.name}.gradle.kts"
 }
+
+buildless {
+  remoteCache {
+    enabled = extra.properties["remoteCache"] == "true"
+    push.set(extra.properties["cachePush"] == "true")
+  }
+}
+
+buildCache {
+  local {
+    isEnabled = true
+    removeUnusedEntriesAfterDays = 14
+    directory = file(".codebase/build-cache")
+  }
+}
```

Comment:
Remote caching is off by default; local caching cleans itself every 14 days and stores local artifacts in `.codebase/build-cache`. Thus, doing a full clean/reset/clone etc. will drop your build cache, too.